baseapp/abci_utils.go

```diff
@@ -112,19 +119,14 @@ func ValidateVoteExtensions(
 			return fmt.Errorf("failed to verify validator %X vote extension signature", valConsAddr)
 		}
 
-		sumVP = sumVP.Add(bondedTokens)
+		sumVP += vote.Validator.Power
 	}
 
-	// Ensure we have at least 2/3 voting power that submitted valid vote
-	// extensions.
-	totalVP, err := valStore.TotalBondedTokens(ctx)
-	if err != nil {
-		return fmt.Errorf("failed to get total bonded tokens: %w", err)
-	}
-
-	percentSubmitted := math.LegacyNewDecFromInt(sumVP).Quo(math.LegacyNewDecFromInt(totalVP))
-	if percentSubmitted.LT(VoteExtensionThreshold) {
-		return fmt.Errorf("insufficient cumulative voting power received to verify vote extensions; got: %s, expected: >=%s", percentSubmitted, VoteExtensionThreshold)
+	if totalVP > 0 {
```

Comment:
Not sure if this should ever be possible, but adding this as a sanity check.

Reply:
its totalVP a signed integer? lmfao 🤣

I'd flip the logic around, personally. <= 0

Reply:
lol better safe than sorry...

Reply:
kek